Sources/StreamChatUI/ChatMessageList/Attachments/FileAttachmentViewInjector.swift

```diff
@@ -42,3 +42,15 @@ private extension FilesAttachmentViewInjector {
         contentView.content?.fileAttachments ?? []
     }
 }
+
+public class MixedAttachmentViewInjector: AttachmentViewInjector {
+    lazy var injectors = [GalleryAttachmentViewInjector(contentView), FilesAttachmentViewInjector(contentView)]
```

Comment:
Attachment catalog from `Components` should be used here, otherwise, custom file/gallery injectors provided by the customer would be ignored

Reply:
@evsaev Good catch! 👍 This is one catch that deserves request changes 😛

Reply:
Fixed ✅